airflow/providers/mysql/CHANGELOG.rst

```diff
@@ -27,6 +27,14 @@ used with MySQL server 5.6.4 through 5.7.
 Changelog
 ---------
 
+4.0.2
+.....
+
+Misc
+~~~~
+
+* ``Use MariaDB client binaries in arm64 image for support MySQL backend (#29519)``
```

Comment:
I wonder if we should reword this. The provider specific change was simply to set the dependencies regardless of arch.

Reply:
Oh, well that's a serious bummer. Commit messages aren't necessarily good changelog entries. We should look at loosening that to look for the PR number instead to correlate.

I don't think it's worth doing anything extra for this case.

Reply:
We know we want to move to newsfragment... just no one had the time to make it happen yet

Reply:
Yep, for sure. I was just thinking an easy interim solution.